src/CoreWCF.Http/tests/Async767311Tests.cs

```diff
@@ -47,19 +47,25 @@ public void Variation_EndMethod()
         }
 
         [Fact]
-        public void Variation_WaitMethod()
+        public async Task Variation_WaitMethod()
         {
             IWebHost host = ServiceHelper.CreateWebHostBuilder<Startup>(_output).Build();
             using (host)
             {
-                host.Start();
+                await host.StartAsync();
                 System.ServiceModel.BasicHttpBinding httpBinding = ClientHelper.GetBufferedModeBinding();
                 var factory = new System.ServiceModel.ChannelFactory<IClientAsync_767311>(httpBinding, new System.ServiceModel.EndpointAddress(new Uri($"http://localhost:{host.GetHttpPort()}/BasicWcfService/SyncService.svc")));
                 IClientAsync_767311 clientAsync_ = factory.CreateChannel();
                 _output.WriteLine("Testing [Variation_WaitMethod]");
                 IAsyncResult asyncResult = clientAsync_.BeginEchoString(clientString, null, null);
                 _output.WriteLine("Message sent via Async, waiting for handle to be signaled");
-                asyncResult.AsyncWaitHandle.WaitOne();
+                
+                // Convert WaitOne() to async polling
```

Comment:
The purpose of this test is to validate the use of AsyncWaitHandle, so revert the polling back to using the AsyncWaitHandle.